tier1/checklist.md

```diff
@@ -0,0 +1,387 @@
+| Getting started       | Included installations steps, prerequisites for installation, and instructions for working with the source code                                                                                                                                                                                     |              |
+| Building dependencies | This step is often skipped, so don't forget to include the steps needed to install on your platform. If your project can be multi-platform, this is an excellent place for first time contributors to send patches!                                                                                 |              |
+| Building the Project  | Be sure to include build scripts and instructions, not just the source code itself!                                                                                                                                                                                                                 |              |
+| Writing Issues        | Make a brief statement about where to file issues, and conventions for doing so.                                                                                                                                                                                                                    |              |
+| Policies              | This section is here to explicitly link to Federal policies and guidelines that are required or recommended for Federal projects to comply with, such as Accessibility (508) Interoperability, Anti-deficiency, Security, Licensing, and other policies that can vary between agencies and domains. |              |
+| Public Domain         | This section is to explicitly link to Federal policies and guidelines that required or recommended for Federal projects to comply with, such as Accessibility (508) Interoperability, Anti-deficiency, Security, Licensing, and other policies that can vary between agencies and domains.          |              |
+
+- [ ] **repolinter.json**
+_Lints repository for missing files and sections above_
+
+**Results**
+*Insert review here*
+
+
+----
+
+
+### Review Repository Details
+
+The GitHub repository homepage features a concise description of the project, a list of relevant topic tags, and general information about the repository to provide a comprehensive overview for users and contributors.
+
+_About Section_:
+
+- [ ] *Description*
+  1-2 sentences describing the project
+
+- [ ] *Website*
+Link to project's website
+
+- [ ] *Topics*
+Tags for project discoverability. Helpful topics to classify a repository include the repository's intended purpose, subject area, community, or language.
+
+_Include in Home Page_:
+
+- [ ] Releases
+- [ ] Packages
+- [ ] Deployments
+
+
+
+**Results**
+
+*Insert review here*
+
+
+
+----
+
+
+
+### Additional Notes & Questions
+
+"*Insert any notes or questions here*"
```

Comment:
No quotes here

Reply:
Remove quotes
"*Insert any notes or questions here*" -> *Insert any notes or questions here*

Reply:
I do not see this on my side, it seems to be already fixed